plugins/custom_commands.py

```diff
@@ -14,7 +14,9 @@ class CustomCommands(BasePlugin):
     default_config = {
         "cc_file": "config/ccs.json",
         "default": {
-            "cc_prefix": "!!"
+            "cc_prefix": "!!",
+            "cc_create_ban": [],
```

Comment:
Storing this in config is *okay*, but the shelve storage is more suitable for this sort of thing.